src/app/components/share/share.js

```diff
@@ -0,0 +1,28 @@
+import {Controller} from 'superb';
+import {description as template} from './share.html';
+
+/* eslint-disable */
+(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.8";
```

Comment:
@edwoodward This is a Facebook-provided snippet to load their SDK. The parameters d, s, and id refer to the supplied arguments document, 'script', and 'facebook-jssdk'.